docs/docs/chitchat-faqs.mdx

```diff
@@ -2,336 +2,72 @@
 id: chitchat-faqs
 sidebar_label: Chitchat and FAQs
 title: Chitchat and FAQs
+abstract: FAQ assistants are the simplest assistants to build and typically the first kind of assistant anyone builds. This page is a guide to the concepts and training data you need to handle non-contextual questions like FAQs and chitchat.
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
+[FAQs](./glossary.mdx#faqs) and [chitchat](./glossary.mdx#chitchat) are two cases where the conversational assistant's answer is a single message,
+and the assistant should always answer the same way,
```

Comment:
hm, it's not really a requirement that the assistants answer is a single message, it can also be multiple. the only requirement is that the user only sends one message.